pallets/did/src/lib.rs

```diff
@@ -409,20 +409,20 @@ pub mod pallet {
 				DidError::StorageError(storage_error) => Self::from(storage_error),
 				DidError::SignatureError(operation_error) => Self::from(operation_error),
 				DidError::InputError(input_error) => Self::from(input_error),
-				DidError::InternalError => Self::InternalError,
```

Comment:
Why are we not refactoring the above ones? Eg getting rid of the _Error_ suffix for ServiceError, StorageError etc.